### PR TITLE
RS-243 Create log table

### PR DESCRIPTION
**Pull request by l-payan**
Add helper to centralize all logs functions
Create Log table and add entries

### CODE REVIEW

File: lib/db_logger.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      request_payload: params&.except(:controller, :action),
+      message: msg,
+      has_error: has_error
+    }
+
+    binding.pry
+
+    Log.create!(data)
+  end
+end
```
Comment (clopezh) [inline lib/db_logger.rb]: Add a new blank line at the end of the file please 🍹

**Review by clopezh — CHANGES_REQUESTED** [review]
Hi @l-payan Looks awesome!! the only comment I have is if is possible to add the user id in each log, to determine which user is doing what

File: lib/db_logger.rb (hunk 1 above)
Comment (l-payan) [reply lib/db_logger.rb]: Done!

**Comment by l-payan** [comment]
Hi @clopezh, I already handle that on the db_logger, is using the @current_user that is generated in application controller
image

**Review by clopezh — APPROVED** [review]

**Comment by clopezh** [comment]
Hey @l-payan good job! but the PR has a merge conflict, Could you please solve it and then I will merge it!

**Comment by l-payan** [comment]
@clopezh Done! You can merge it now

**Pull request merged**